pyop2/base.py

```diff
@@ -256,17 +254,15 @@ def __init__(self, data=None, map=None, idx=None, access=None, flatten=False):
            defined on.
 
         A :class:`MapValueError` is raised if these conditions are not met."""
-        self._dat = data
+        self.data = data
```

Comment:
I assume there's a good reason for renaming (which is orthogonal to performance)?